docs/reference/jspsych-pluginAPI.md

```diff
@@ -507,8 +507,7 @@ None.
 
 #### Description
 
-Generates a `MediaRecorder` object from provided `MediaStream` and stores this for access via [`getCameraRecorder()`](#getcamerarecorder).
-
+Generates a `MediaRecorder` object from provided `MediaStream` and stores this for access via [`getCameraRecorder()`](#getcamerarecorder). By default, `mimeType` is set to `"video/webm"`.
```

Comment:
Minor suggestion to change the last sentence to something like this:
"By default, `mimeType` is set to  the first compatible container/codec combination found in a list of common types, or just `"video/webm"` if no supported combination is found."